src/main/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceAdapter.java

```diff
@@ -432,7 +433,13 @@ public VAppTemplate getImage(final String imageId) {
                  .transform(new Function<Reference, VApp>() {
                     @Override
                     public VApp apply(Reference in) {
-                       return api.getVAppApi().get(in.getHref());
+                       try {
+                          return api.getVAppApi().get(in.getHref());
+                       } catch (VCloudDirectorException e) {
+                          // If the VApp is in an inconsistent state, the server will return at 500 error and an
+                          // exception will be thrown, in which case we can only skip this VApp
+                          return null;
```

Comment:
Please log the exception, rather than throwing it away. This code would likely bite us at some point in the future when there is a different reason for the exception!